Packs/AzureSentinel/Scripts/MicrosoftSentinelSubmitNewComment/MicrosoftSentinelSubmitNewComment.yml

```diff
@@ -0,0 +1,27 @@
+args:
+- name: new_comment
```

Comment:
lets also add an optional argument here for incident id. When not provided get from the calling context

Reply:
Done